### PR TITLE
Added rendering of tank in armory

### LINKED ISSUE #190: Armory: Render selected tank in Armory (medium)
**Describe the solution you'd like**
We currently aren't rendering the selected tank in the armory. We should be able to do this by creating a new ImageDrawer(src/battleground/ImageDrawer.js) but passing in a different canvas to draw on (here we would pass in a new canvas in the armory. We should still be able to use the same Tank class that we are using to battle with, we just wouldn't call Tank.executePhysics(), so it wouldn't like walk around the screen on us or anything.

If we wanted to, one thing we could do that would be really cool imo is have some custom Casus code that just animates the tank to turn the turret left, wait for a couple seconds, turn it back to the right, wait a couple seconds, et cetera. If we have this code saved, it would be really easy to put it into the tank and run it every 20th of a second, and then we would have a pretty idle animation for our tank when people are building their own tank.

### PR DESCRIPTION
**Pull request by SecondThread**
**Description**
Made tanks render in the armory. See #190 for details

Resolves #190

**Type of change**
Check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Fix or feature that would cause existing functionality to not work as expected
- [ ] This change requires a update in the design document

**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
Looks like this:
image

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Pulled updated master branch (if changed since branch creation) and corrected any conflicts, if any occur.

### CODE REVIEW

File: src/tanks/TankDisplay.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import Vec from '../casus/blocks/Vec.js';
+import {imageLoaderInit, addCallbackWhenImageLoaded} from '../battleground/ImageLoader.js';
+import './TankDisplay.css';
+
+type Props = {|
+	tankToDisplay: Tank
+|};
+
+type State = {|
+|};
```
Comment (JorgeVidal29) [inline src/tanks/TankDisplay.js]: Does state have to be here?

File: src/tanks/TankDisplay.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+		const canvas: HTMLCanvasElement = this.refs.canvas;
+		const targetWidth=canvas.clientWidth;
+		const targetHeight=targetWidth*8/16;
+		if (canvas.width !== targetWidth || canvas.height !== targetHeight) {
+			canvas.width = targetWidth;
+			canvas.height = targetHeight;
+		}
+	}
+
+	_lerp(a: number, b: number, time: number) {
```
Comment (JorgeVidal29) [inline src/tanks/TankDisplay.js]: return type

File: src/tanks/TankDisplay.js (hunk 1 above)
Comment (SecondThread) [reply src/tanks/TankDisplay.js]: nope, thanks for catching

File: src/tanks/TankDisplay.js (hunk 2 above)
Comment (SecondThread) [reply src/tanks/TankDisplay.js]: Indeed, nice catch

**Review by JorgeVidal29 — APPROVED** [review]
Looks good to me

**Pull request merged**